bibliopixel/animation/off.py

```diff
@@ -2,11 +2,11 @@
 
 
 class OffAnim(BaseAnimation):
-    """A trivial animation that turns all LEDs off."""
+    """A trivial animation that turns all LAYOUTs off."""
```

Comment:
A layout is singular, which has multiple pixels. Should be "turns all pixels in layout off"